application/src/main/java/org/togetherjava/tjbot/commands/tags/TagManageCommand.java

```diff
@@ -306,10 +310,11 @@ private void handleActionWithMessage(@NotNull TagStatus requiredTagStatus,
      * @param id the id of the tag to get its content
      * @return the content of the tag, or {@code "Unable to retrieve content"} if was unable to
```

Comment:
adjust now incorrect javadoc

Reply:
is this good?
```
/**
 * Gets the content of a tag.
 *
 * @param subcommand the subcommand to be executed
 * @param id the id of the tag to get its content
 * @return the content of the tag, if was able to get it
 */
```

Reply:
yeah. usually people write `the content of the tag, if present` and thats enough already, since the type is optional so people understand.

Reply:
so just?
```
/**
 * Gets the content of a tag.
 *
 * @param subcommand the subcommand to be executed
 * @param id the id of the tag to get its content
 * @return the content of the tag, if present
 */
```

Reply:
yes

Reply:
no problem. done.